rest_framework/utils/model_meta.py

```diff
@@ -80,16 +81,16 @@ def get_field_info(model):
 
 def _get_pk(opts):
     pk = opts.pk
-    while pk.rel and pk.rel.parent_link:
+    while get_remote_field(pk) and get_remote_field(pk).parent_link:
         # If model is a child via multi-table inheritance, use parent's pk.
-        pk = pk.rel.to._meta.pk
+        pk = get_remote_field(pk).to._meta.pk
```

Comment:
Could we instead have this set in a var after line 83? `remote_field = get_remote_field(pk)` and use throughout?